ts/Extensions/DashboardPlugins/DataGridComponent.ts

```diff
@@ -193,6 +194,32 @@ class DataGridComponent extends Component<DataGridComponent.ChartComponentEvents
                     this.store.table.setColumns(e.table.getColumns());
                 }
             });
+
+            // Update the DataGrid when store changed.
+            this.store.table.on('afterSetCell', (e: any): void => {
+                const dataGrid = this.dataGrid;
+                let shouldUpdateTheGrid = true;
+
+                if (dataGrid) {
+                    const row = dataGrid.rowElements[e.rowIndex],
+                        cells = Array.prototype.slice.call(row.childNodes);
+
+                    cells.forEach((cell: HTMLElement): void => {
+                        if (cell.childElementCount > 0) {
+                            const input =
+                                cell.childNodes[0] as HTMLInputElement;
+
+                            if (cell.dataset.columnName === e.columnName &&
+                                +input.value === e.cellValue
+                            ) {
+                                shouldUpdateTheGrid = false;
+                            }
+                        }
+                    });
```

Comment:
Good job. 👍 It is quite some overhead. Let's do this for now. Maybe in the future we should check, if we can save and check the `table.version` property.